templates/admin.php

```diff
@@ -90,6 +90,13 @@
 					<span><?php p($l->t('days until user password expires'));?></span>
 				</label>
 			</li>
+			<li>
+				<label>
+					<input type="checkbox" name="spv_user_password_expiration_notification_checked" <?php if ($_['spv_user_password_expiration_notification_checked']): ?> checked="checked"<?php endif; ?>/>
+					<input type="number" name="spv_user_password_expiration_notification_value"  min="0" value="<?php p($_['spv_user_password_expiration_notification_value']) ?>" placeholder="30"/>
+					<span><?php p($l->t('days to send a notification before the password expires'));?></span>
```

Comment:
this message sounds odd, maybe "number of days until expiration after which to send a notification"

@phil-davis any better ideas ?

Reply:
"send a notification this many days before the password expires"
If it is OK to use this style of grammar.
Forcing the sentence so that it begins with the number "30" makes the English difficult, whatever you do.